engine/benches/bench.rs

```diff
@@ -242,6 +244,41 @@ fn bench_string_function_comparison(c: &mut Criterion) {
+    ];
```

Comment:
I doubt it's a good idea to have this in the public code

Reply:
Sure, i'll provide a PR quickly to change that.

Reply:
You might want to remove it from history altogether before merging if it's a sensitive information.

Reply:
We have other projects pointing at commits from this branch so rewriting history will forbid us to build older versions of our internal tools unless we keep/fork this branch internally. I don't believe its sensitive information but i'll check. Also, that's unfortunate but other people might have already checked out / forked this branch so I am afraid we won't be able to remove it from the internet anyway :-(